helpers/filterSeats.js

```diff
@@ -21,8 +24,11 @@ module.exports = function(dust) {
     var venueProduct = _.head(params.venueProducts);
     var ticketRates = params.ticketRates;
     var roomRates = params.roomRates;
-    var bandColours = params.bandColours;
-    var sortByInput = params.sortBy;
+    var sortType = params.sortBy;
```

Comment:
We've renamed sortByInput to sortType, but it's still using sortBy as a param. This might be confusing? Can we rename the param to match this?